game-state-service

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+from gi.repository import Gio
+from gi.repository import GLib
+from gi.repository import GObject
+
+import dbus
+import dbus.service
+import dbus.mainloop.glib
```

Comment:
Using dbus-glib and dbus-python is a bit of a red flag for me. We should use GIO and GDBus for all new services we build.